cogs/utility.py

```diff
@@ -104,7 +104,8 @@ async def user_info(self, ctx, *, user: discord.User = None):
 
         embed.add_field(name=f"{self.bot.settings['emoji']['shadows']} Username", value=f"{db_user['fullUsername']}"
                                                                                         f" {' '.join(badges)}")
-        embed.add_field(name=f"{self.bot.settings['emoji']['id']} ID", value=db_user["_id"])
+        # embed.add_field(name=f"{self.bot.settings['emoji']['id']} ID", value=db_user["_id"]) # You can change this idc
+        embed.set_footer(text=f"ID: db_user["_id"]")
```

Comment:
nothing really wrong with this, but I will probably get others thoughts first

Reply:
it basically reformats what was already there

Reply:
That's what you call invalid syntax and incorrect f-string. It'd actually be f"ID: {db_user['_id']}" _ik this is closed but still letting you know cause why not._